### ISSUE TITLE
Release 1.9.0 available without tag (might be compromised/hijacked)

### ISSUE THREAD

**Issue by gaaf** [go-mysql-org/go-mysql#908]
Hi,

I noticed that [pkg.go.dev](https://pkg.go.dev/github.com/go-mysql-org/go-mysql) serves a version 1.9.0 of this package. On the github repo there's no such [tag](https://github.com/go-mysql-org/go-mysql/tags), the changelog doesn't mention a 1.9.0 version and the links on said page are dead.

I cannot determine if the served package is legitimate. Please either update the changelog and tag the appropriate commit _or_ make sure pkg.go.dev doesn't serve an unauthorized version.

**Comment by dveeden** [comment]
Related: https://github.com/go-mysql-org/go-mysql/pull/906

**Comment by dveeden** [comment]
Looks like https://github.com/go-mysql-org/go-mysql/pull/907 is also related

**Comment by dveeden** [comment]
cc @atercattus

**Comment by atercattus** [comment]
Yes, this is my mistake. I make a new tag 1.9 without checking that [main branch is working](https://github.com/go-mysql-org/go-mysql/actions/runs/10466023616/job/28982182945). When I realized this, I removed the tag ASAP to avoid distributing unstable main. Unfortunately, pkg.go.dev already saw the new tag...

We are thinking about these fixes here https://github.com/go-mysql-org/go-mysql/pull/907

**Comment by lance6716** [comment]
I have recreated v1.9.0 https://github.com/go-mysql-org/go-mysql/tags let's see if it can overwrite the old tag

**Comment by lance6716** [comment]
I think I should create v1.9.1 https://go.dev/blog/go116-module-changes

> a version cannot be modified after it is published
